tutorials/asr/09_ASR_for_telephony_speech.ipynb

```diff
@@ -0,0 +1,423 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import librosa\n",
+    "import IPython.display as ipd\n",
+    "import librosa.display\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# Load and listen to the audio file\n",
+    "example_file = data_dir + '/an4/wav/an4_clstk/mgah/cen2-mgah-b.wav'\n",
+    "audio, sample_rate = librosa.load(example_file)\n",
+    "\n",
+    "audio_16kHz = librosa.core.resample(audio, sample_rate, 16000)\n",
+    "print(sample_rate)\n",
+    "import numpy as np\n",
+    "\n",
+    "# Get spectrogram using Librosa's Short-Time Fourier Transform (stft)\n",
+    "spec = np.abs(librosa.stft(audio_16kHz))\n",
+    "spec_db = librosa.amplitude_to_db(spec, ref=np.max)  # Decibels\n",
+    "\n",
+    "# Use log scale to view frequencies\n",
+    "librosa.display.specshow(spec_db, y_axis='log', x_axis='time', sr=16000)\n",
```

Comment:
Everywhere sample rate is 16 kHz, so limitting ylim to 8k doesnt achieve much no?

Reply:
8K imposed by ylimit is the max frequency, for 16 kHz it is 8kHz
![Screen Shot 2021-06-15 at 2 08 44 AM](https://user-images.githubusercontent.com/4916480/122027852-50cc3400-cd80-11eb-83e2-e5cc64d47a7d.png)
 but for 8 kHz audio there are no frequencies above 4kHz
![Screen Shot 2021-06-15 at 2 08 51 AM](https://user-images.githubusercontent.com/4916480/122027945-63df0400-cd80-11eb-9c74-b27a034c9de6.png)

Reply:
Ah interesting, didnt realize that.